File: java/src/main/java/itp/itp1_3/ITP1_3_B.java
package itp.itp1_3;

import java.io.PrintWriter;
import java.util.Scanner;

public class ITP1_3_B {

  public static void main(String[] args) {
    try (
      Scanner sc = new Scanner(System.in);
      PrintWriter pw = new PrintWriter(System.out)) {
      int n = 1;
      
      while (true) {
        int x = sc.nextInt();

        if (x == 0) break;

        pw.append("Case " + n++ + ": " + x + "\n");
      }
      pw.flush();
    }
  }
}

File: kotlin/src/itp/itp1_3/ITP1_3_B.kt
package itp.itp1_3

import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    var i = 1
    while (true) {
        val x = sc.nextInt()
        if (x == 0) break
        println("Case ${i}: $x")
        i++
    }
}